treeherder/log_parser/parsers.pyx

```diff
@@ -314,6 +314,7 @@ RE_ERR_MATCH = re.compile((
     r"|^abort:"
     r"|^Output exceeded \d+ bytes"
     r"|^The web-page 'stop build' button was pressed"
+    r".*\\.js: line \\d+, col \\d+, Error - .*"
```

Comment:
RE_ERR_MATCH is built as one long list of OR'd regexes, so this needs the pipe symbol at the start.
Also, in Python regexes defined using string literals ('r' prefix), we don't need to double escape the backslashes.
We can also drop the trailing ".*", since its implicit with .match()